### PR TITLE
Adds update_gce_image command

### PR DESCRIPTION
**Pull request by robertabbott**
Testing: Updated an image with and without
passing the --encrypted-image-name option
and verified the new image booted

### CODE REVIEW

File: brkt_cli/__init__.py [hunk 1]
```diff
@@ -217,11 +219,38 @@ def command_launch_gce_image(values, log):
...
+
+    if not encrypted_image_name:
+        nonce = util.make_nonce()
+        # Replace nonce in image name
+        name = values.image
+        m = re.match('(.+) \(encrypted (\S+)\)', name)
+        if m:
+            encrypted_image_name = m.group(1) + '-encrypted-%s' % (nonce,)
+        else:
+            encrypted_image_name = name + '-encrypted-%s' % (nonce,)
```
Comment (bburtin) [inline brkt_cli/__init__.py]: Is there a limit on the length of image names in GCE?

File: brkt_cli/update_gce_image.py [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+        updater = instance_name + '-metavisor'
+        encrypted_image_disk = instance_name + '-guest'
+
+        # Create disk from encrypted guest snapshot. This disk
+        # won't be altered it will be re-snapshotted and paired
+        # with the new encrypted image.
+        gce_svc.disk_from_snapshot(zone, image_id, encrypted_image_disk)
+        gce_svc.wait_for_disk(zone, encrypted_image_disk)
+        log.info("Creating snapshot of encrypted image disk")
+        gce_svc.create_snapshot(zone, encrypted_image_disk, encrypted_image_name)
```
Comment (bburtin) [inline brkt_cli/update_gce_image.py]: Does this snapshot get cleaned up if the process aborts?

File: brkt_cli/update_gce_image.py [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+        log.info('Deleting updater instance')
+        gce_svc.delete_instance(zone, updater)
+
+        # wait for updater root disk
+        gce_svc.wait_for_detach(zone, updater)
+
+        # create image from mv root disk and snapshot
+        # encrypted guest root disk
+        log.info("Creating updated metavisor image")
+        gce_svc.create_gce_image_from_disk(zone, encrypted_image_name, updater)
```
Comment (bburtin) [inline brkt_cli/update_gce_image.py]: Does this image get cleaned up if the process fails?

Comment (robertabbott) [inline brkt_cli/update_gce_image.py]: Added some checks to make sure everything gets deleted

File: brkt_cli/__init__.py (hunk 1 above)
Comment (robertabbott) [inline brkt_cli/__init__.py]: the name is now truncated if > 63 characters

**Comment by bburtin** [comment]
Thanks Bobby.  Looks good.  Merge away!

**Pull request merged**